tools/c7n_gcp/c7n_gcp/resources/spanner.py

```diff
@@ -44,6 +45,37 @@ def get_label_params(resource, all_labels):
                         'field_mask': ', '.join(['labels'])}}
 
 
+@resources.register('spanner-instance-backup')
```

Comment:
s/spanner-instance-backup/spanner-backup

Reply:
could you provide more details? should we add aliases?

Reply:
I think just change the resource name to `spanner-backup` which matches what gcp does for it in asset inventory.

Reply:
fixed